Gemfile

```diff
@@ -1,47 +1,30 @@
 source 'https://rubygems.org'
 
-
```

Comment:
I wouldn't worry about removing these comments. It shouldn't significantly affect performance and it's nice to have these explanations for those who're unfamiliar with these gems.